tests/001-test-use-devs-to-create-thin-pool.sh

```diff
@@ -17,6 +17,7 @@ test_devs() {
   cat << EOF > /etc/sysconfig/docker-storage-setup
 DEVS="$devs"
 VG=$vg_name
+CONTAINER_THINPOOL=docker-pool
```

Comment:
Then the fixes we did yesterday will not take effect except in the weird case where they specify the a argv1 and argv2 that does not have a CONTAINER_THINPOOL defined, but they previously created a docker-pool thinpool.